config/config.exs

```diff
@@ -20,6 +20,6 @@ config :logger, :console,
 
 # Import environment specific config. This must remain at the bottom
 # of this file so it overrides the configuration defined above.
-import_config "#{Mix.env}.exs"
-import_config "amqp.exs"
 import_config "datagouvfr.exs"
+import_config "amqp.exs"
```

Comment:
Any reason to change the order ?

Reply:
So we can override the configuration with the environment configuration.